Documentation/ApiOverview/BackendModules/CliScripts/Index.rst

```diff
@@ -54,8 +54,9 @@ A simple example can be found in the :php:`ListSysLogCommand`:
     /**
      * Configure the command by defining the name, options and arguments
      */
-    public function configure()
+    protected function configure()
     {
+        parent::configure();
```

Comment:
This line is unusual and [actually not required](https://symfony.com/doc/current/console.html#configuring-the-command).